### ISSUE TITLE
actionCreator returns action, not a function

### ISSUE THREAD

**Issue by mbohgard** [orourkek/typesafe-redux#2]
@orourkek Typings is correct, but this example (from your todo app example) returns an action with a `undefined` payload, not a function.

```
export const editTodo = actionCreator('EDIT_TODO')<{
  id: ITodo['id'],
  newText: ITodo['text']
}>();
```

**Comment by orourkek** [comment]
@mbohgard I'm not sure exactly what you mean, as the typings and usage in the example looks correct to me. Are you seeing an `undefined` payload in your built code (js)?

**Comment by mbohgard** [comment]
Yes @orourkek, typings are correct, actual return from the actionCreator is an action, not a function.

**Comment by orourkek** [comment]
This is fixed starting with version 2.0.0
